src/ralph/models/xapi/fields/verbs.py

```diff
@@ -11,28 +15,42 @@
 
 
 class ViewedVerbField(BaseModelWithConfig):
-    """Represents the `verb` xAPI Field for page viewed xAPI statement.
+    """Represents the `verb` xAPI Field for the action `viewed`.
 
     Attributes:
        id (str): Consists of the value `http://id.tincanapi.com/verb/viewed`.
        display (dict): Consists of the dictionary `{"en": "viewed"}`.
     """
 
-    id: VERB_VIEWED_ID = VERB_VIEWED_ID.__args__[0]
-    display: dict[LANG_EN_DISPLAY, VERB_VIEWED_DISPLAY] = {
-        LANG_EN_DISPLAY.__args__[0]: VERB_VIEWED_DISPLAY.__args__[0]
+    id: Literal[VERB_VIEWED_ID] = VERB_VIEWED_ID
+    display: dict[Literal[LANG_EN_US_DISPLAY], Literal[VERB_VIEWED_DISPLAY]] = {
+        LANG_EN_US_DISPLAY: VERB_VIEWED_DISPLAY
     }
 
 
 class TerminatedVerbField(BaseModelWithConfig):
-    """Represents the `verb` xAPI Field for page terminated xAPI statement.
+    """Represents the `verb` xAPI Field for the action `terminated`.
 
     Attributes:
        id (str): Consists of the value `http://adlnet.gov/expapi/verbs/terminated`.
        display (dict): Consists of the dictionary `{"en": "terminated"}`.
     """
 
-    id: VERB_TERMINATED_ID = VERB_TERMINATED_ID.__args__[0]
-    display: dict[LANG_EN_DISPLAY, VERB_TERMINATED_DISPLAY] = {
-        LANG_EN_DISPLAY.__args__[0]: VERB_TERMINATED_DISPLAY.__args__[0]
+    id: Literal[VERB_TERMINATED_ID] = VERB_TERMINATED_ID
+    display: dict[Literal[LANG_EN_US_DISPLAY], Literal[VERB_TERMINATED_DISPLAY]] = {
+        LANG_EN_US_DISPLAY: VERB_TERMINATED_DISPLAY
+    }
+
+
+class SavedVerbField(BaseModelWithConfig):
```

Comment:
xAPI verb definitions are quite similar, only the default `id` and `display` values are different, may be we could use a factory here ? Example: [verbs.py](https://github.com/SergioSim/ralph/blob/build-verb-field/src/ralph/models/xapi/fields/verbs.py)

Reply:
I prefer being declarative and thus repetitive for our models. Having factories will end up with an unmaintainable spaghetti code.

Reply:
Thanks) Indeed, this would be harder to maintain)